res/Sectors/M1105/Ricenden.txt

```diff
@@ -0,0 +1,466 @@
+
+# Name: Ricenden (hv)
+# Name: Gnoghikt! (kk)
+
+# Milieu: M1105
+
+# Credits: Ricenden sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Giostuxe
+# Subsector B: Nuana
+# Subsector C: Noonol
+# Subsector D: Osstaet
+# Subsector E: Etdirkorneer
+# Subsector F: Theeka
+# Subsector G: Phleskfan
+# Subsector H: Bluaskup
+# Subsector I: Hive
+# Subsector J: Kont
+# Subsector K: Iytganornim
+# Subsector L: Phlesige
+# Subsector M: Omeq
+# Subsector N: Lyarysklyixana
+# Subsector O: Luhawro
+# Subsector P: Eamlammaxaar
+
+# Alleg: Hf: "Hive FDA Uplift/Observation Site"
+# Alleg: HvFd: "Hive Federation"
+# Alleg: Kr: "Krotan Primacy"
+# Alleg: CsHv: "Hiver Client State"
+# Alleg: Na  : "Non-Aligned, Hiver-dominated"
+
+Hex  Name                 UWP       Remarks                      {Ix}   (Ex)    [Cx]   N B  Z PBG W  A    Stellar   
+---- -------------------- --------- ---------------------------- ------ ------- ------ - -- - --- -- ---- -------------------------
+0101 Osest                C544340-5 Lo                           { -2 } (621-1) [7155] - N  - 800 11 Kr   A2 V            
+0102 Ubwix                E4527CD-6 Po                           { -2 } (764+2) [A514] - -  - 913 16 Kr   M8 V M2 V       
+0104 Muveng               B8A5740-A Fl                           {  2 } (B6E+2) [694A] - -  - 322 13 CsHv M7 III          
+0206 Prostyskem           E431217-6 Lo Po (Prostyskem) Pz        { -3 } (711-2) [3137] - E  A 902  6 HvFd M1 V K1 V      
+0210 Tiinfel              A7599S6-F Hi                           {  4 } (A8C-2) [CD2F] - KM - 802  7 HvFd M5 V M8 V M2 V  
+0303 Vrojuggus            E88A756-8 Ri Wa                        { -1 } (A66+1) [4648] - -  - 123 11 CsHv M4 V            
+0305 Wrorudat             C547203-5 Lo                           { -2 } (A11-1) [1169] - -  - 714 13 HvFd M8 V            
+0308 Ketcaqi              C4828T6-9 Ri Ph                        {  1 } (67D+3) [A949] - M  - 300  9 HvFd A3 V            
+0309 Dururyrt             B888503-7 Ag Ni Pr                     {  0 } (842+4) [3598] - -  - 600  8 HvFd M9 V M7 V       
+0310 Nezzummi             B326105-B Lo IthkW                     {  1 } (B01-1) [3217] - K  - 803 15 HvFd F5 V            
+0401 Klaslob              C682753-9 Ri                           {  1 } (96C+5) [5849] - N  - 321 15 Kr   G3 V M0 V       
+0405 Arfanafglant         B897405-B Ni Pa                        {  1 } (B36+2) [458B] - -  - 525 16 HvFd M3 V            
+0406 Fostdoorir           D2519U2-8 Hi Po                        { -1 } (786+0) [9847] - -  - 712 11 HvFd M9 V            
+0407 Naxdeldensk          E100000-0 Ba Va                        { -3 } (600-3) [0000] - -  - 014 14 HvFd M8 V            
+0408 Giostuxe             A75AA25-D Cp Hi Wa                     {  3 } (796-1) [9D4D] - K  - 600  7 HvFd M2 V M3 V       
+0501 Wa'abjak             A659854-B Ph                           {  2 } (97A+0) [7A2C] - N  - 710 14 Kr   M8 V            
```

Comment:
This should have a 'Cx' code (Capital)

Reply:
The capital of the Krotan Primacy is meant to be Emyzi, I've just mislabelled that as Cp. I corrected this, and made this world a Cp instead